depends/patches/zeromq/cmake_minimum.patch

```diff
@@ -0,0 +1,18 @@
+Set a more sane cmake_minimum_required.
+
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,12 +1,7 @@
+ # CMake build script for ZeroMQ
++cmake_minimum_required(VERSION 3.16)
```

Comment:
Unless this patch is submitted upstream, why not use the same version as our CMake-based build system?

Reply:
Reverting this, otherwise this will just be blocked by the CI (without other changes that I'm not making here), and matching the CMake version isn't a requirement.